rank/.prettierrc.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
   "singleQuote": true,
-  "semi": false
-}
\ No newline at end of file
+  "semi": false,
+  "tabWidth": 2
```

Comment:
[The default is already 2](https://prettier.io/docs/en/options.html#tab-width)

Reply:
think this was introduced while I was trying to resolve the linting problems i mentioned above ☝️